programs/graph_program/src/state/connection.rs

```diff
@@ -1,17 +1,37 @@
-use anchor_lang::prelude::*;
+use anchor_lang::{prelude::*};
+
+#[derive(AnchorSerialize, AnchorDeserialize, Clone, PartialEq, Eq)]
+pub enum ConnectionStatus {
+    Connected = 1,
+    Disconnected = 2,
+}
 
 #[account]
 #[derive(Default)]
 pub struct Connection {
     pub from: Pubkey,
     pub to: Pubkey,
+    // V2 Items.
+    pub status: Option<ConnectionStatus>, // No connection status equals connected
+    pub connected_at: Option<i64>, // No connected_at time equals connected
```

Comment:
connected_at doesn't need to be optional. if connection exists it must have a connected_at. disconnected_at optional.

Reply:
What happens with existing accounts?

Reply:
Added migration instruction